### PR TITLE
feat: Add RSS reader links

### LINKED ISSUE #37: Add RSS reader subscription links
Add a link to RSS readers for subscribing the target feed.

### PR DESCRIPTION
**Pull request by takuan-osho**
## Overview

This PR adds open-in-reader links to discovered feed results so users can hand off a feed URL to common hosted RSS readers. The UI intentionally opens reader deep links rather than claiming that Feed Finder directly completes a subscription inside the user's reader account.

The reader options are exposed as an accessible disclosure with plain external links. Feed URLs are validated before link generation, encoded into fixed reader origins, and disambiguated in accessible labels so duplicate feed titles remain usable.

A second commit fixes the pre-push security-audit blocker by overriding vulnerable transitive versions of ws and brace-expansion. Direct Cloudflare package versions are unchanged.

## Related Issue

Closes #37

## Changes

- Add RSS reader deep-link generation for Feedly, Inoreader, NewsBlur, The Old Reader, and Feedbin
- Add an "Open in reader" action to each feed result card
- Use accessible disclosure/region semantics instead of menu roles
- Validate feed URLs so only http/https URLs produce reader links
- Add tests for encoding, unsafe schemes, duplicate titles, keyboard close behavior, and external-link safety
- Add dependency overrides to clear the pre-push audit blocker

## Impact Scope

This affects the feed results UI and test/development dependency resolution. Feed discovery, worker APIs, and deployment configuration are unchanged.

## Checklist

- [x] Code follows the style guidelines
- [x] Tests have been added/updated
- [x] Documentation has been updated (if relevant)
- [x] Build is successful
- [x] Ready for review

## How to Test

- npm run test:run
- npm run lint
- XDG_CONFIG_HOME=/private/tmp/feed-finder-xdg npm run build
- npm audit

Pre-push hooks also passed during branch push: biome, security-audit, and typecheck.

## Additional Information

The reader links open third-party subscription/add-feed flows. They do not authenticate to reader APIs or directly mutate a user's reader account.

## Summary by CodeRabbit

* **New Features**
  * フィードカードに「RSSリーダーで開く」オプションを追加し、Feedly・Inoreader・NewsBlur・The Old Reader・Feedbin で直接開けるようになりました。オプションは折りたたみ可能で、Escapeで閉じフォーカスが戻ります。

* **Chores**
  * 一部依存パッケージのバージョン指定を追加・調整しました。

* **Tests**
  * RSSリーダー連携や表示・キーボード操作の動作を検証するテストを追加しました。

[Review Change Stack](https://app.coderabbit.ai/change-stack/takuan-osho/feed-finder/pull/60?utm_source=github_walkthrough&utm_medium=github&utm_campaign=change_stack)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,6 +53,8 @@
     "wrangler": "^4.85.0"
   },
   "overrides": {
+    "brace-expansion": "^5.0.6",
+    "ws": "^8.20.1",
```
[the remark this answers is not in the document]
Comment (takuan-osho) [reply package.json]: Updated the ws override to require 8.21.0 or newer. I kept the other
overrides as ranges to avoid turning temporary security overrides into
unnecessary long-term pins; the lockfile still records the resolved versions
for reproducible installs.

File: src/lib/feedReaders.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    label: "Feedbin",
+    buildUrl: (feedUrl: string) =>
+      `https://feedbin.com/?subscribe=${encodeURIComponent(feedUrl)}`,
+  },
+] as const;
+
+export function getFeedReaderLinks(feedUrl: string): FeedReaderLink[] {
+  if (!isHttpFeedUrl(feedUrl)) {
+    return [];
+  }
```
[the remark this answers is not in the document]
Comment (takuan-osho) [reply src/lib/feedReaders.ts:44]: この指摘は妥当ではないと判断したため、採用しません。

`getFeedReaderLinks` は同期的な UI 表示用ヘルパーであり、不正な feed URL はユーザーへ表示するエラーではなく「リーダーリンクを表示しない」ための条件です。そのため、`Result` を返して呼び出し側にエラー処理を強制するより、空配列を返す現在の挙動の方がこの責務には合っています。
